.editorconfig

```diff
@@ -10,9 +10,14 @@ insert_final_newline = true
 indent_style = tab
 indent_size = 4
 
-[{*.json,*kml,*.md}]
+[{.*,*.json,*.md}]
 indent_style = space
 indent_size = 2
 
-[*.csv]
+[site/includes/sitemenureplace.hbs]
+insert_final_newline = false
+indent_style = space
+indent_size = 1
+
+[src/i18n/i18n.csv]
```

Comment:
Think it makes sense to keep the general setting for `*.csv` since i guess it's true that trailing space can be significant in CSV files

Reply:
In that case - I'll revert this part back to its previous state.

FWIW, the second CSV file contains quotes around all of its strings and doesn't currently contain any trailing spaces.